topics/single-cell/tutorials/scrna-case_trajectories/tutorial.md

```diff
@@ -0,0 +1,446 @@
+> 3. **Rename** {% icon galaxy-pencil %} output h5ad `T-cell_object.h5ad`
+>
+{: .hands_on}
+
+You should now have `8569` cells, as opposed to the `8605` you started with. You've only removed a few cells (the contaminants!), but it makes a big difference in the next steps.
+
+## Draw force-directed graph
+
+First, we will calculate a [force-directed graph](https://scanpy.readthedocs.io/en/stable/api/scanpy.tl.draw_graph.html) (FDG), as an alternate to tSNE, which will likely work better for trajectory analysis.
+
+> <hands-on-title> Draw FDG </hands-on-title>
+>
+> 1. {% tool [Scanpy RunFDG](toolshed.g2.bx.psu.edu/repos/ebi-gxa/scanpy_run_fdg/scanpy_run_fdg/1.8.1+galaxy9) %} with the following parameters:
+>    - {% icon param-file %} *"Input object in AnnData/Loom format"*: `T-cell_object.h5ad` (output of **Manipulate AnnData** {% icon tool %})
+>    - *"Use programme defaults"*: {% icon history-share %} `No`
+>    - *"Graph layout"*: `fa`
+>
+>    > <comment-title> Graph Layout </comment-title>
+>    >
+>    > We're using the fa or ForceAtlas2 layout for our FDGs. It is the same layout used in the [Jupyter notebook version of this tutorial]({% link topics/single-cell/tutorials/scrna-case_JUPYTER-trajectories/tutorial.md %}) and works well for our data. As well as choosing the fa layout when we create the FDGs, we will also specify the `draw_graph_fa` embedding when drawing the plots. 
+>    {: .comment}
+> 
+{: .hands_on}
+
+## Plot the FDG
+
+And now time to plot it!
+
+> <hands-on-title> Plot the FDG </hands-on-title>
+>
+> 1. {% tool [Scanpy PlotEmbed](toolshed.g2.bx.psu.edu/repos/ebi-gxa/scanpy_plot_embed/scanpy_plot_embed/1.8.1+galaxy9) %} with the following parameters:
+>    - {% icon param-file %} *"Input object in AnnData/Loom format"*: `FDG object Anndata` (output of **Scanpy RunFDG** {% icon tool %})
+>    - *"name of the embedding to plot"*: `draw_graph_fa`
+>    - *"color by attributes, comma separated texts"*: `cell_type`
+>    - *"Use raw attributes if present"*: `No`
+>    - *"Location of legend"*: `On data`
+{: .hands_on}
+
+
+> <question-title></question-title>
+>
+> 1. What has the FDG done to our clusters of T-cells and what might this suggest about the relationships between these groups?
+>
+> > <solution-title></solution-title>
+> >
+> > 1. Well now this is exciting! Our DP-late is more clearly separating, and we might also suppose that DP-M1, DP-M2, and DP-M3 are actually earlier on in the differentiation towards mature T-cells. And we're only just getting started!
+> >
+> > ![Plot showing the T Mat and DP L clusters have separated from the rest of the cells](../../images/scrna-case_trajectories/TrajectoriesFDG1.png "FDG Plot of our T-cell dataset")
+> {: .solution}
+>
+{: .question}
+
+## Diffusion maps
+
+We'll now perform an *optional step*, that basically takes the place of the PCA. Instead of using PCs, we can use [diffusion maps](https://scanpy.readthedocs.io/en/stable/api/scanpy.tl.diffmap.html).
```

Comment:
```suggestion
We'll now perform an *optional step*, that basically takes the place of the standard Principle Component Analysis (PCA). Instead of using PCs, we can use [diffusion maps](https://scanpy.readthedocs.io/en/stable/api/scanpy.tl.diffmap.html).
```